chromadb/test/property/strategies.py

```diff
@@ -89,13 +103,13 @@ def one_or_both(strategy_a, strategy_b):
 int_types = [np.int16, np.int32, np.int64]  # TODO: handle int types
 
 
-@st.composite
-def collection_name(draw) -> str:
+@st.composite  # type: ignore
```

Comment:
@levand These are ignored everywhere, should they instead have some sort of type hint? if so, what ?

Reply:
Try merging and this might go away from our update to pre commit config